README.md

```diff
@@ -1 +1,162 @@
+  <h3><b>Math Magicians setup</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Math Magicians <a name="about-project"></a>
+
+**Math magicians** Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+-Make simple calculations.
+-Read a random math-related quote.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+  <li><a href="#">HTML</a></li>
+  <li><a href="#">CSS</a></li>
+    <li><a href="#">React vite</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Make Math calculation**
+- **Read a random math-related quote**
+<p align="right">
+(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- comming soon
+
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+  ## 💻 Getting Started <a name="getting-started"></a>
+
+  To get a local copy up and running it in your machine, follow the next steps.
+
+  ### Prerequisites
+
+  In order to run this project you need:
+
+- Internet connection.
+- Basic knowledge of Git/Terminal.
+- Visual Studio Code Installed.
+
+### Setup
+
+Clone this repository to your desired folder
+
+### Install
+
+Install almost all dependencies.
+This one will install ESLint:
+
+> npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x @babel/plugin-syntax-jsx@7.x @babel/preset-react@7.x @babel/preset-react@7.x
+
+This one will install Stylelint:
+
+> npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### Usage
+
+To run the project,clone the project from https://github.com/haftamudesta/Math-Magicians
+
+> use the command git checkout component to go to the component dirctory
+> to create react use npm create vite@latest -- --template react
+> to install all raect dependencies use npm install after installing reat
+> to run the code use npm run dev
+
```

Comment:
- [ **OPTIONAL** ] To make your documentation be more professional and descriptive. I think it would be better if you improve al the above section by adding all information needed by users in order to `clone` your project, `access` the clone directory, `install` dependencies, `run tests` (if exist),... To help you, I suggest you refer this **[example](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)**